ETC.tsv

```diff
@@ -15916,23 +15916,23 @@ ETC_20151016_015917	Silver Pillar
 ETC_20151016_015918	Bronze Pillar
 ETC_20151016_015919	Officer Lutas
 ETC_20151016_015920	Defeat the Gray Golem and Carapace!
-ETC_20151022_015921	Yellow Collifly
+ETC_20151022_015921	Yellow Colifly
 ETC_20151022_015922	Evil Necroventer
-ETC_20151022_015923	Rafemaiter is an unique occupation that receives the respect and the insult at the same time among Rafes. Rafemaiter serves a religious role, but other Rafes don't put great emphasis on religious rituals so they want Rafemaiter to do the fatigue duty.
-ETC_20151022_015924	Unlike the prejudice of the humans, Rafejads are not Rafes that obatined the current status through long training. Most of them get selected by their breeds and they grow up accordingly and receive the necessary food to become Rajejads.
-ETC_20151022_015925	Their status is not high among Rafes, but they are most dangerous and they get encountered often. To receive respect from other Rafes, they act violently to the ones they encounter with.
-ETC_20151022_015926	For a Rafe to Raflammer, they just need to obtain the weapons they need. But its not that easy since when Raflammers die, the weapons also get destroyed so its not that easy for Rafes.
-ETC_20151022_015927	Ferret Archers usually engages in battles, but they sometimes shoot down the fruits on tall trees. It's not that other Ferrests asked them to do it, but they just do that for fun when they are not engaged in battles.
+ETC_20151022_015923	Lapemiter is an unique occupation that receives the respect and the insult at the same time among Lapes. A Lapemiter serves a religious role, but the other Lapes don't put great emphasis on religious rituals so they want Lapemiter to do their exhausting duty.
```

Comment:
Lapemiter is a unique occupation that is highly respected, but is also looked down upon among Lapes.  A Lapemiter serves a religious role, but the other Lapes don't put great emphasis on religious rituals, so they want a Lapemiter to do their exhausting duty.